docs/topics/wasm/wasm-overview.md

```diff
@@ -1,61 +1,75 @@
-> If you face any issues, please report them on [GitHub](https://github.com/JetBrains/compose-multiplatform/issues).
->
-{type="note"}
+We regularly run benchmarks on Kotlin/Wasm, and these results come from our testing in a recent version of Google Chrome.
+
+## Browser API support
+
+The Kotlin/Wasm standard library provides declarations for browser APIs (including the DOM API).
+With these declarations, you can directly use the Kotlin API to access and utilize various browser functionalities, like 
+manipulation with DOM elements or the use of fetch API, in your Kotlin/Wasm applications without defining these declarations 
+from scratch. To learn more, see our [Kotlin/Wasm browser example](https://github.com/Kotlin/kotlin-wasm-examples/tree/main/browser-example).
 
-Compose Multiplatform for Web is based on new Kotlin/Wasm target.
-You can create a Kotlin Multiplatform project and experiment with sharing your mobile or desktop UIs with the web.
-With Compose Multiplatform for Web, you can run your code in the browser with all the benefits of WebAssembly.
+The declarations for browser API support are defined using JavaScript [interoperability capabilities](wasm-js-interop.md). 
+You can use the same capabilities to define your own declarations. In addition, Kotlin/Wasm and Kotlin/JS interoperability allows you to use 
+Kotlin code from JavaScript. For more information, see [Use Kotlin code in JavaScript](wasm-js-interop.md#use-kotlin-code-in-javascript).
 
-## How to get started
 
-* [Get started with Kotlin/Wasm in IntelliJ IDEA](wasm-get-started.md)
-* Check out the [GitHub repository with Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples)
 
-## Libraries support
+## Leave feedback
 
-You can use the Kotlin standard library (`stdlib`) and test library ([`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/))
-in Kotlin/Wasm out of the box. The version of these libraries is the same as the version of the `kotlin-multiplatform` plugin.
+### Kotlin/Wasm
+* ![Slack](slack.svg){width=25}{type="joined"} Slack: provide your feedback directly to developers in our [#webassembly](https://kotlinlang.slack.com/archives/CDFP59223) channel. [Get a Slack invite here](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
+* ![Youtrack](youtrack-black-logo.png){width=25}{type="joined"} Youtrack: report any issues in [YouTrack](https://youtrack.jetbrains.com/issue/KT-56492).
 
-Kotlin/Wasm has an experimental support for other Kotlin libraries. [Read more how to enable them in your project](wasm-libraries.md).
+### Compose Multiplatform for Web
+* ![Slack](slack.svg){width=25}{type="joined"} Slack: provide your feedback in the [#compose-web](https://slack-chats.kotlinlang.org/c/compose-web) public channel.
+* ![Github](git-hub.svg){width=25}{type="joined"} Github: report any issues
+  [here](https://github.com/JetBrains/compose-multiplatform/issues).
```

Comment:
We don't use "here" as a part of the link. Maybe it's better to use "report any issues" as a link?